### DISCUSSION TITLE
How to update Thorium?

### DISCUSSION THREAD

**Discussion by Arenimo** [Alex313031/thorium#908] [Q&A]
Hello,

Im a Thorium user; with very poor tech-knowledge. How do I update Thorium? I have read a few things about this topic; I saw a tool called winupdater but it also says support-ended.
How do you update the browser? I am currently using Version 128.0.6613.189. 
Thanks.

**Comment by zenmarz** [comment]
There is any new update is not released from thorium

**Comment by brambora69123** [comment]
There is currently no thorium update yet as of writing this. You can watch the releases section of the repository for any new updates.

**Comment by vippium** [comment]
> Hello,
> 
> Im a Thorium user; with very poor tech-knowledge. How do I update Thorium? I have read a few things about this topic; I saw a tool called winupdater but it also says support-ended. How do you update the browser? I am currently using Version 128.0.6613.189. Thanks.

You can simply update, when Alex release a updated build for it.
I think you'll be auto-updated to newer version after released, as Alex implemented some sort of updater in last releases. But I suggest to keep an eye on releases page.
